core/src/com/unciv/logic/civilization/TechManager.kt

```diff
@@ -25,20 +25,6 @@ class TechManager {
     @Transient
     private var researchedTechUniques = ArrayList<Unique>()
 
-    // MapUnit.canPassThrough is the most called function in the game, and having these extremely specific booleans is or way of improving the time cost
```

Comment:
I think we should split policy and tech efforts by use Uniques from policymanager and techmanager, like you do now.